app/controllers/concerns/remote_record_controller.rb

```diff
@@ -0,0 +1,53 @@
+module RemoteRecordController
```

Comment:
If this is just shared behavior between our various curation concern controllers, can we handle this with
```ruby
class CurationConcernController < ApplicationController
end
class ScannedBooksController < CurationConcernController
end
```

rather than multiple inheritance?